.vib/influxdb/goss/vars.yaml

```diff
@@ -0,0 +1,15 @@
+binaries:
+  - influx
+  - influxd
+  - gosu
+directories:
+  - mode: "0775"
+    paths:
+      - /opt/bitnami/influxdb/etc
+      - /bitnami/influxdb/data
+      - /bitnami/influxdb/wal
+      - /bitnami/influxdb/meta
+root_dir: /opt/bitnami
+version:
+  bin_name: influx
```

Comment:
Somewhat of a nitpick, knowing that `influx` refers to the CLI binary and `influxd` to the InfluxDB, we should use the second one here.